### ISSUE TITLE
We need to create GMLC test app

### ISSUE THREAD

**Issue by nhanth87** [RestComm/gmlc#10]
GMLC is important in 4G network and we should create test app to test new GMLC feature

**Comment by FerUy** [comment]
Indeed. The Geolocation project we're into is counting on it, not only for 4G (LTE) but for improved location within SS7 networks in 3G Radio Access Network (UMTS Terrestrial RAN). It's important to differentiate between the two, as for 4G it must follow Diameter extensions for SLg (GMLC-MME) and SLh (GMLC-HSS), as per 3GPP TS 29.172 annd 3GPP 29.173 respectively, while for 3G it only demands adding two MAP operations (SRI for Location Services or SRIforLCS, and Provide Subscriber Location or PSL), of course specified by 3GPP TS 29.002 (MAP).
The Restcomm Geolocation API we're building at this stage of the project has all of the aforementioned considered implicitly, as it will trigger Geolocation services in the GMLC.

**Comment by FerUy** [comment]
@nhanth87 @vetss this is the issue I was talking about today in the conference call after checking with @abhayani that locationSvcGatewayContext and locationSvcEnquiryContext Application Context Name operations including MAP SRIforLCS, MAP PSL and MAP SLR (Subscriber Location Report), are already implemented (see attached, MAP Op Codes 83, 85, 86
[MAPMessagesImplemented.xlsx](https://github.com/RestComm/gmlc/files/245615/MAPMessagesImplemented.xlsx)
). Hence, Lg + Lh interfaces with MSCS and HSS respectively would be available for testing. This is tied to #1 issue opened by @deruelle and assigned to myself, as part of the Geolocation project.
![restcomm_gmlc umts](https://cloud.githubusercontent.com/assets/14883482/14962443/28ab9344-1075-11e6-8c36-10956cb7781d.png)

**Comment by angrygreenfrogs** [comment]
@FerUy I'm not totally clear about this issue - is this covered by the jss7 simulator, unit tests, and documentation, and so isn't really an issue by itself?

**Comment by FerUy** [comment]
@angrygreenfrogs, yeah, this issue's comments are misleading, my bad mainly as Tran opened it for LTE only.
It's in part covered by jSS7 simulator (and ongoing work there as noted above, main reason why I don't close it as it is a reference for that), but also there will be another test application to be developed for Diameter as for #2 (in which I'm already working).
